subo/release/release.mk

```diff
@@ -0,0 +1,5 @@
+now = $(shell date +'%Y-%m-%dT%TZ')
+commit = $(shell if [ ! -d .git ]; then echo "unknown"; else git rev-parse HEAD; fi)
```

Comment:
That Git command outputs the "full" commit SHA, is there a way to use the shortened version (I think it's the prefix?)